model_bakery/baker.py

```diff
@@ -1,15 +1,19 @@
+from __future__ import annotations
```

Comment:
This import seems to be unnecessary since it's not being used and it's breaking CI tests as well.

Reply:
This line changes the behaviour of type annotations, allowing for postponed evaluation (for example, allowing `def create(...): -> Baker[_NewM]` method to refer to it's own class).
https://docs.python.org/3/whatsnew/3.7.html#pep-563-postponed-evaluation-of-annotations

Reply:
But I see that does break 3.6 compatibility. Fixing...